.github/actions/argus-builder/build-prep/action.yml

```diff
@@ -21,6 +21,10 @@ inputs:
     description: 'Branch names to run this job on, supports wildcards, comma delimited'
     required: false
     default: ''
+  labels:
+    description: 'Labels attached to an associated PR'
+    requred: false
+    default: ''
```

Comment:
rename to something that indicates these are the labels currently attached to the PR. Then we need another input for the name of the label with the default value being "stack". I think we should allow for the name of the label to be configured, with a sane default

Reply:
done